.github/workflows/spelling.yml

```diff
@@ -55,6 +55,7 @@ jobs:
         id: spelling
         run: |
            python -m pyspelling > misspellings.txt || true
+           python ci_tools/json_aspell.py misspellings.txt
            cat misspellings.txt
            python ci_tools/summarise_pyspelling.py misspellings.txt $GITHUB_STEP_SUMMARY
```

Comment:
i would add my functions to ``summarise_pyspelling.py``, for testing purposes and to not break anything i did create a new file, ill add the proper flags to generate the json file

Reply:
I suspect it's easier to test in `summarise_pyspelling.py` because it means you don't need to touch the workflow which means even the bot can run the test. If you break something on a branch it doesn't matter. That's why I turned off commits to master ;)